test/verify/check-dashboard

```diff
@@ -318,11 +319,7 @@ class TestDashboardSetup(MachineCase, DashBoardHelpers):
         self.assertIn("docker", groups(m2, "junior"))
         self.assertIn(m2.get_admin_group(), groups(m2, "senior"))
 
-        try:
-            m2.execute("id nosync")
-            self.assertFalse("User nosync should not have been synced")
-        except:
-            pass
+        self.assertRaises(subprocess.CalledProcessError, lambda: m2.execute("id nosync"))
```

Comment:
Do it like this instead to avoid wrapping and unwrapping the lambda;

    self.assertRaises(subprocess.CalledProcessError, m2.execute, "id nosync")

Reply:
Didn't know that was possible, thanks.